src/install/python/python-agent-intro.mdx

```diff
@@ -0,0 +1,30 @@
+---
+componentType: default
+---
+
+import pythonagentWebTransactionChart from 'images/apm-python-agent_screenshot-crop_web-transaction-chart.png'
+
+
+Do you want to monitor the performance of your Python application? You can do this by installing our Python agent! It is a powerful tool that collects telemetry about your application and then forwards that data to New Relic. Once the data is in New Relic, you can view a wide range of UI pages that give you insights into what's happening with your software.
+
+
+    <img
+      title="Screenshot showing web transaction for Python agent"
+      alt="Screenshot showing web transaction for Python agent"
+      src={pythonagentWebTransactionChart}
+    />
+    <figcaption>
+      This is an example of the type of chart you would see in the New Relic UI if you installed the Python agent
+    </figcaption>
+
+## Start our interactive instructions
+
+This page is interactive, which means we'll ask you some questions about your environment, and then we'll give you some specific steps to install the Python agent. If you change your mind after making some selections, click **Reset the form** to start again:
```

Comment:
I still feel kind of leery about this reset button, but since it's working, I guess we might as well try it out.